File: Sender/Sender.py
import stomp
# stomp.Connection([('127.0.0.1', 62613)])
conn = stomp.Connection10()

conn.start()

conn.connect()

while True:
    lusho = input("Consola Python: ")
    if lusho == "salir":
        break
    conn.send('DonL', lusho)
    print("Mensaje enviado")
conn.disconnect()

File: Sender/Sender.js
"use strict";
const stompit = require('stompit');
const readline = require('readline');

const rl = readline.createInterface({
    input: process.stdin,
    output: process.stdout
});

stompit.connect({ host: 'localhost', port: 61613 }, (err, client) => {
    const frame = client.send({ destination: 'SampleQueue' });

    rl.question('Consola Javascript: ', (answer) => {
        if (answer == "salir") return;
        frame.write(answer);
        frame.end();
        console.log('Mensaje enviado');

        rl.close();
    });

    client.disconnect();
});